connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConnectorTest.java

```diff
@@ -86,31 +90,8 @@ public void testInitializeFailure() {
         RuntimeException exception = new RuntimeException();
         connector = sourceConnector;
 
-        connector.version();
-        expectLastCall().andReturn(VERSION);
-
-        offsetStore.start();
```

Comment:
I am adding this comment here to help other reviewers in reviewing the changes in these lines.

It is ok to remove these void calls because Mockito automatically creates stubs for void method calls. As long as we are verifying that they are being invoked, we should be good.